tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/UserEvents/IIdentityUser.cs

```diff
@@ -2,6 +2,7 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
+
 using System.ComponentModel;
 
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.UserEvents;
```

Comment:
Should we give ownership to the ASM team on the UserEvents folder? May not make senses if we have to impact the aspnetcore integration each time we touch them.

Reply:
Yes I think the 3 integrations will have to be modified, if we have to make changes to the user events.. :/